fix(Logs Kinesis Firehose): CF-10272 (#5826)
newrelic/docs-website: src/content/docs/logs/forward-logs/stream-logs-using-kinesis-data-firehose.mdx

  **Note**: To send your logs to the EU, complete the remaining steps in this section, then proceed to the [configuration procedures for EU accounts](#configure-eu-stream).

7. Paste your [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key) in the **API key** field.
8. Configure and review the remaining metadata settings.

> Any optional key/value pairs you add in the AWS Management Console will result in attribute/value pairs you can use in New Relic.

Reviewer: I was a bit thrown by this sentence. How about something more like? 

```suggestion
Any optional key/value pairs you add to the AWS Management Console create attribute/value pairs you can use in New Relic.
```
